editor/app/src/main/java/ca/ubc/eml/soiltopargraphy/editor/ui/flag/JFlag.java

```diff
@@ -0,0 +1,74 @@
+package ca.ubc.eml.soiltopargraphy.editor.ui.flag;
+
+import ca.ubc.eml.soiltopargraphy.editor.ui.infopanel.JInfoPanel;
+
+import java.util.ArrayList;
+
+
+
+public class JFlag {
+
+    private String location;
+    private float lattitude;
+    private float longditude;
+    private ArrayList<JInfoPanel> infoPanels;
+    private int id;
+
+
+    public JFlag(String location, float lattitude, float longditude, ArrayList<JInfoPanel> infoPanels, int id) {
```

Comment:
Once you delete the original Flag.kt class, change this JFlag to "Flag" and JInfoPanel to InfoPanel.